config/routes.js

```diff
@@ -259,6 +259,16 @@ module.exports.routes = {
     action: 'harvest',
     csrf: false
   },
+  'post /:branding/:portal/api/mint/harvest/:recordType': {
+    controller: 'webservice/RecordController',
+    action: 'legacyHarvest',
+    csrf: false
+  },
+  'get /:branding/:portal/api/mint/harvest/search': {
```

Comment:
We only need the ingest harvest. Search will be performed with existing endpoints already covered in the first work package. Future work package will cover getting the form components that search legacy Mint to be able to find the records inside ReDBox.